tests/ctst/steps/iam-policies/IAMUser.ts

```diff
@@ -0,0 +1,50 @@
+import { Given, setDefaultTimeout } from '@cucumber/cucumber';
+import { Constants, IAM, Utils } from 'cli-testing';
+
+setDefaultTimeout(Constants.DEFAULT_TIMEOUT);
+
+/**
+ * Policy are using ARN, not names. This helper will dynamically extract a policy
+ * ARN from a CLI result
+ * @param {object} results - results from the command line
+ * @return {string} - the policy arn, or null if an error occured when
+ * parsing results.
+ */
+function extractPolicyArnFromResults(results: any) {
+    try {
+        if (results.stdout) {
+            return JSON.parse(results.stdout).Policy.Arn;
+        }
+        return null;
+    } catch (err) {
+        return null;
+    }
+}
+
+Given('an existing IAM policy with {string} effect to perform {string} on {string}', async function (effect: string, action: string, resource: string) {
+    this.endForType();
+    this.resetCommand();
```

Comment:
Maybe you can create a helper for these 2 actions, if we want to run them in multiple steps, it will allow to centralize place where we reset things, if needed